recipes/strong_type/all/conanfile.py

```diff
@@ -0,0 +1,50 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class StrongtypeConan(ConanFile):
+    name = "strong_type"
```

Comment:
```suggestion
    name = "enhex-strong_type"
```

Reply:
I'm not familiar with the package_info() renaming stuff.
can you provide an example?

Reply:
package_info is not needed for your recipe, as it's header-only. The suggestion is because `strong_type` is too generic and a fresh project, it could collide with other projects like https://github.com/doom/strong_type

Reply:
i get it that part, but the part about `package_info` `.name` `.filename` and `set_property` is unclear, and searching the Conan docs didn't help much.

Reply:
> but the part about package_info .name .filename and set_property is unclear, and searching the Conan docs didn't help much.

It's not required for your recipe. To get more information about, please, read:

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info
https://docs.conan.io/en/latest/reference/generators/cmake_find_package_multi.html#generated-files

Reply:
It's not strictly required unless you want it to keep generating files and targets without the extra `enhex-` prefix (after renaming the package to `enhex-strong-type`).